### PR TITLE
feat: Fixes #6839 show mfa shared secret

### LINKED ISSUE #6839: feat: MFA TOTP should display a seed code to enter into an application that does not use QR Codes
If an application that generates TOTP does not have a QR scanner a seed code should be displayed on the TOTP user screen in the MFA Management screen.

Most TOTP systems will display both the QR code and the seed value.  I ran into this problem when trying to test the login screen with MFA.

### PR DESCRIPTION
**Pull request by adunsulag**
In order to use a non-qr type TOTP code we need to show the shared secret when going to add the code.

Fixes #6839

### CODE REVIEW

File: interface/usergroup/mfa_totp.php [hunk 1]
```diff
@@ -185,6 +185,8 @@ function docancel() {
                                             <br />
                                             <img src="<?php echo attr($qr); ?>" class="img-responsive center-block" style="height:200px !Important"/>
                                             <br />
+                                            <p><?php echo xlt("Or paste in the following code into your authenticator app"); ?></p>
+                                            <p><?php echo $mfaAuth->getSecret(); ?></p>
```
Comment (bradymiller) [inline interface/usergroup/mfa_totp.php:189]: also should do at https://github.com/openemr/openemr/blob/master/setup.php#L1377-L1404
(this allows setting up mfa on the user during openemr installation)

Comment (adunsulag) [reply interface/usergroup/mfa_totp.php:189]: @bradymiller Thanks for catching that.  Added to the setup.

Comment (sjpadgett) [reply interface/usergroup/mfa_totp.php:189]: Yep I call him ole Eagle Eye! 😉

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV-2(2)

**Pull request merged**